src/malawi/get_floodscan_data.py

```diff
@@ -9,19 +9,25 @@
 
 path_mod = f"{Path(os.path.dirname(os.path.realpath(__file__))).parents[1]}/"
 sys.path.append(path_mod)
+
+from src.indicators.flooding.config import Config
 from src.indicators.flooding.floodscan import floodscan
 
 COUNTRY_NAME = "malawi"
+config = Config()
+parameters = config.parameters(COUNTRY_NAME)
+COUNTRY_ISO3 = parameters["iso3_code"].lower()
```

Comment:
Is there a reason you are retrieving the country_iso3 here while it seems it is not being used?